### ISSUE TITLE
How to use colab-katago 

### ISSUE THREAD

**Issue by hope366** [yzyray/lizzieyzy#11]
How do I use colab-katago with lizzieyzy?
It works with normal lizzie. I used the exact same engine command as normal lizzie, but got an error. 
![無題](https://user-images.githubusercontent.com/63999713/140538397-c0a1222b-e37f-4303-97c2-1da5384afe67.png)

**Comment by wonsiks** [comment]
@hope366 It seems that we can't use colab-katago any more.
I think Google has blocked katago from using OpenCL resource on colab.
However you may use engine command as follows :
"c:\users\...\colab-katago-gd.exe" 1pSlMjPR7qsOfhAq2aR_zxkkxxxxxxxxxxx 12345678
This can make lizzieyzy connect to colab but katago can't run analysis properly.

**Comment by hope366** [comment]
@wonsiks It seems that someone familiar with python has found a solution. 
At the end of the code to install katago, adding 4 lines like this seems to work. 
```
# Copy enviroment variable for SSH
with open('/root/.ssh/environment','w') as f:
    for key, value in os.environ.items():
        print('{}={}'.format(key, value), file=f)

!echo "done!"
```
I still got the error in my environment, but I succeeded by adding `import os`. 
It seems that all you have to do is place it where similar code is lined up. 

Now colab-katago works with regular lizzie and Sabaki. But still lizzieyzy gives an error. 
I first wrote the engine command as follows: 
`"C:\\Users...\\colab-katago-gd.windows\\colab-katago-gd.exe xxxxxxxx xxxx"`
This didn't work, so I then tried the following command that you told me. 
`"C:\\Users...\\colab-katago-gd.windows\\colab-katago-gd.exe" xxxxxxxx xxxx`
But that didn't work, and I got the error shown above. 
Do you get the above error in your environment? Do I need to fill in information about ssh when setting up the engine?

**Comment by wonsiks** [comment]
@hope366 Oh, by the code you gave colab works again. Thanks...
And you may setup engine command from setup dialog rather than from config.txt file.
Here I give my setting..
<img width="666" alt="20211106_114912" src="https://user-images.githubusercontent.com/37356450/140595534-1ee619d4-0f2d-4304-94e6-bd4acde134e2.png">
.

**Comment by hope366** [comment]
@wonsiks Thank you. Thanks to your help, I was able to get colab-katago running on lizzieyzy. 
I used to copy the engine commands used in normal lizzie as they are, but this seems to have been the cause of the failure.
As you say, setting it from the lizzieyzy engine setup dialog worked fine. 
In normal lizzie,`\\` is sometimes used at the boundary between folders, but copying this to lizzieyzy as it is was the cause of the failure.
